packages/@react-native-community/changelog-generator/src/changelog-generator.ts

```diff
@@ -41,7 +41,7 @@ export const CHANGES_TEMPLATE: Changes = Object.freeze(
 ) as Changes;
 
 const CHANGELOG_LINE_REGEXP = new RegExp(
-  `(\\[(${[...CHANGE_TYPE, ...CHANGE_CATEGORY].join("|")})\\]\s*)+`,
+  `(\\[(${[...CHANGE_TYPE, ...CHANGE_CATEGORY].join("|")})\\]\\s*)+`,
```

Comment:
In response to `no-usless-escpaes`. I'm guessing we want the whitespace here instead of a literal "s" character, so I added another backslash.